package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@figma/plugin-typings",
-  "version": "1.34.0",
```

Comment:
I think we always bump the second number here (34). Not sure if there's a good reason for that though. Since this one does break existing plugin parameters, maybe it's worth making this a full update.

Reply:
I didn't want to bump the second number since I wasn't planning on making an announcement on the update "blog", and having a missing number there seems confusing

Reply:
Sounds good to me